rest/doc_api_test.go

```diff
@@ -161,4 +161,15 @@ func TestGuestReadOnly(t *testing.T) {
 	response = rt.SendRequest("PUT", "/db/doc?rev=1-ca9ad22802b66f662ff171f226211d5c", `{"val": "newval"}`)
 	assertStatus(t, response, http.StatusForbidden)
 
+	// Attempt to access _blipsync as guest - blip sync handling for read-only GUEST is applied at replication level (to allow pull-only replications).
+	// Should succeed permission check, and only fail on websocket upgrade
+	response = rt.SendRequest("GET", "/{{.db}}/_blipsync", "")
```

Comment:
I don't think these `{{.db}}`/`{{.keyspace}}` things will work on old branches. They were added for collections.

older branches had hardcoded db names